### ISSUE TITLE
Additional header

### ISSUE THREAD

**Issue by NgKX1641** [worktile/ngx-gantt#451]
Hello, I see there is PrimaryDatePoints and SecondaryDatePoints for headers, is there a way for me to add a new header?
I am trying to display the year, week, day of week, and date at the same time.

**Comment by walkerkay** [comment]
Currently not possible, you can try using a custom view.

You can try PrimaryDatePoints to display both year and week, and secondaryDatePoints to display both week and date. However, if that's the case, you need to set the cellWidth in ViewOptions to be larger.

https://gantt.ngnice.com/components/custom-view
https://github.com/worktile/ngx-gantt/tree/master/example/src/app/gantt-custom-view

**Comment by NgKX1641** [comment]
Yes I have thought of this approach, but having year and week in the same field makes year display repeatedly along with every week, so year is just repeating over and over again.

**Comment by walkerkay** [comment]
Currently, only two - level date grouping is supported. To support three - level date grouping, the rendering view would need to be modified. There are no plans for this at the moment.
